src/components/list/list.scss

```diff
@@ -0,0 +1,125 @@
+@import "variables";
+
+$md-list-side-padding: 16px;
+$md-list-avatar-size: 40px;
+
+/*  Normal list variables */
+$md-list-top-padding: 8px;
+$md-list-font-size: 16px;
+$md-list-secondary-font: 14px;
+$md-list-base-height: 48px;
+$md-list-avatar-height: 56px;
+$md-list-two-line-height: 72px;
+$md-list-three-line-height: 88px;
+
+/* Dense list variables */
+$md-dense-top-padding: 4px;
+$md-dense-font-size: 13px;
+$md-dense-base-height: 40px;
+$md-dense-avatar-height: 48px;
+$md-dense-two-line-height: 60px;
+$md-dense-three-line-height: 76px;
+
+@mixin md-list-item-base($font-size, $base-height, $avatar-height, 
+  $two-line-height, $three-line-height) {
+
+  .md-list-item {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    font-family: $md-font-family;
+    box-sizing: border-box;
+    font-size: $font-size;
+    height: $base-height;
+    padding: 0 $md-list-side-padding;
+  }
+
+  &.md-list-avatar .md-list-item {
+    height: $avatar-height;
+  }
+
+  &.md-2-line .md-list-item {
+    height: $two-line-height;
+  }
+
+  &.md-3-line .md-list-item {
+    height: $three-line-height;
+  }
+
+  .md-list-text {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    padding: 0 $md-list-side-padding;
+
+    &:first-child {
+      padding: 0;
+    }
+
+    &:empty {
+      display: none;
+    }
+  }
+
+  [md-list-avatar] {
+    width: $md-list-avatar-size;
+    height: $md-list-avatar-size;
+  }
+}
+
+@mixin md-line-base($secondary-font-size) {
+  * {
```

Comment:
Yeah, I can do that if I move it into the `.md-list-text` section, where it should probably be anyway.  Doing so now...